mk_mingw.mak

```diff
@@ -16,8 +16,8 @@ VPATH = . ./main ./parsers
 ctags.exe: OPT = -O4 -Os -fexpensive-optimizations
 ctags.exe: LDFLAGS = -s
 dctags.exe: OPT = -g
-dctags.exe: DEBUG = -DDEBUG
-dctags.exe: SOURCES += debug.c
+dctags.exe: CFLAGS += -DDEBUG
+# dctags.exe: SOURCES += $(MAIN_DIR)/debug.c
```

Comment:
Could you remove the `#`?

I would say the default behavior of a debug build should be showing debug messages. If you want to disable it yourself and still use the debug build, do so in your personal branch.